server/app/controllers/ti/TrustedIntermediaryController.java

```diff
@@ -134,12 +193,32 @@ public Result addApplicant(Long id, Http.Request request) {
   private Result redirectToDashboardWithError(
       String errorMessage, Form<AddApplicantToTrustedIntermediaryGroupForm> form) {
     return redirect(
-            routes.TrustedIntermediaryController.dashboard(Optional.empty(), Optional.empty()))
+            routes.TrustedIntermediaryController.dashboard(
+                    /* paramName=  nameQuery */
+                    Optional.empty(),
+                    /* paramName=  searchDate */
+                    Optional.empty(),
+                    /* paramName=  page */
+                    Optional.of(1))
+                .url())
         .flashing("error", errorMessage)
-        .flashing("providedFirstName", form.get().getFirstName())
-        .flashing("providedMiddleName", form.get().getMiddleName())
-        .flashing("providedLastName", form.get().getLastName())
-        .flashing("providedEmail", form.get().getEmailAddress())
-        .flashing("providedDob", form.get().getDob());
+        .flashing("providedFirstName", form.value().get().getFirstName())
+        .flashing("providedMiddleName", form.value().get().getMiddleName())
+        .flashing("providedLastName", form.value().get().getLastName())
+        .flashing("providedEmail", form.value().get().getEmailAddress())
+        .flashing("providedDateOfBirth", form.value().get().getDob());
+  }
+
+  private Result redirectToDashboardWithUpdateDateOfBirthError(String errorMessage) {
+    return redirect(
+            routes.TrustedIntermediaryController.dashboard(
+                    /* paramName=  nameQuery */
+                    Optional.empty(),
+                    /* paramName=  searchDate */
+                    Optional.empty(),
+                    /* paramName=  page */
+                    Optional.of(1))
```

Comment:
suggestion (blocking): fix comments to match the style used elsewhere in the codebase